src/librustc_mir/borrow_check/used_muts.rs

```diff
@@ -101,5 +107,7 @@ impl<'visit, 'cx, 'tcx> Visitor<'tcx> for GatherUsedMutsVisitor<'visit, 'cx, 'tc
                 }
             }
         }
+
+        // FIXME: no super_local?
```

Comment:
`super_local` doesn't do anything so this isn't surprising.

Reply:
Doesn't seem right to rely on that, though.

Reply:
Turns out `super_local` does not even exist...^^